Flaky test: tests/chainerx_tests/unit_tests/routines_tests/test_rnn.py test_NStepGru

https://ci.preferred.jp/chainer.py37.cpu/33601/

Occurred in https://github.com/chainer/chainer/pull/8194

> _ test_NStepGru_param_9_{ out_dtype=(), batches=(4, 3, 2), hidden_size=6, in_dtypes=('float16',), input_size=4, n_layers=4}[native:0] _
--
> [FAILED] /chainer/tests/chainerx_tests/unit_tests/routines_tests/test_rnn.py (1 failed, 89 passed, 90 skipped in 17.69 seconds)
--

```
...
2019-10-10 13:16:24.256850 STDOUT 2275] | E           AssertionError: check_double_backward failed (eps=0.001 atol=0.1 rtol=0.1)
-- | --
...
2019-10-10 13:16:29.183954 STDOUT 2275] | E           x: numeric gradient, y: backward gradient
-- | --
2019-10-10 13:16:29.183955 STDOUT 2275] | E           Not equal to tolerance rtol=0.1, atol=0.1
2019-10-10 13:16:29.183955 STDOUT 2275] | E
2019-10-10 13:16:29.183987 STDOUT 2275] | E           Mismatch: 100%
2019-10-10 13:16:29.183993 STDOUT 2275] | E           Max absolute difference: 0.14882182
2019-10-10 13:16:29.184003 STDOUT 2275] | E           Max relative difference: 0.37215233
2019-10-10 13:16:29.184004 STDOUT 2275] | E            x: array(0.251073)
2019-10-10 13:16:29.184005 STDOUT 2275] | E            y: array(0.399895)
2019-10-10 13:16:29.184027 STDOUT 2275] | E
2019-10-10 13:16:29.184029 STDOUT 2275] | E           assert_allclose failed:
2019-10-10 13:16:29.548535 STDOUT 2275] | E             shape: () ()
2019-10-10 13:16:29.548770 STDOUT 2275] | E             dtype: float64 float64
2019-10-10 13:16:29.548772 STDOUT 2275] | E             i: (0,)
2019-10-10 13:16:29.548773 STDOUT 2275] | E             x[i]: 0.25107307469566575
2019-10-10 13:16:29.548774 STDOUT 2275] | E             y[i]: 0.3998948921914566
2019-10-10 13:16:29.548775 STDOUT 2275] | E             relative error[i]: 0.3721523340301626
2019-10-10 13:16:29.548776 STDOUT 2275] | E             absolute error[i]: 0.14882181749579082
2019-10-10 13:16:29.548777 STDOUT 2275] | E             relative tolerance * \|y[i]\|: 0.03998948921914566
2019-10-10 13:16:29.548778 STDOUT 2275] | E             absolute tolerance: 0.1
2019-10-10 13:16:29.548778 STDOUT 2275] | E             total tolerance: 0.13998948921914567
2019-10-10 13:16:29.548779 STDOUT 2275] | E           x: 0.25107307
2019-10-10 13:16:29.548780 STDOUT 2275] | E           y: 0.39989489
```

--- a/tests/chainerx_tests/unit_tests/routines_tests/test_rnn.py
+++ b/tests/chainerx_tests/unit_tests/routines_tests/test_rnn.py
@@ -240,6 +240,7 @@ class TestNStepBiLstm(op_utils.ChainerOpTest):
             'in_dtypes, out_dtype', n_step_lstm_dtypes_valid)
     ])
 ))
+@op_utils.fix_random()  # This test is unstable.
 class TestNStepGru(op_utils.ChainerOpTest):
 
     def setup(self):

